pkg/storage/bucket/s3/config.go

```diff
@@ -88,6 +104,10 @@ func (cfg *Config) Validate() error {
 		return errUnsupportedSignatureVersion
 	}
 
+	if !util.StringsContain(supportedStorageClasses, cfg.StorageClass) {
+		return errUnsupportedStorageClass
```

Comment:
could you please change the error to return the current storage class and all supported storage classes? 
something like : 
`unsupported S3 storage class: users_storage_class . supported storage classes: [STANDARD, STANDARD_IA, ... resto of the classes]`

Reply:
💠